app/controllers/geojson_layers_controller.rb

```diff
@@ -2,7 +2,7 @@ class GeojsonLayersController < ApplicationController
   before_action :set_map, only: [:remove, :add]
   before_action :set_geojson_layer, only: [:show, :edit, :update, :destroy, :remove, :add]
 
-  #layout 'manager'
+  layout 'manager'
```

Comment:
Don't need this with the latest master code, instead have this controller inherit the `ManagerController`